### PR TITLE
Add loot boxes and rarities

### PR DESCRIPTION
**Pull request by mikethetike**
### Migrations
 * No Migrations

### Environment Variables
 * No change

### CODE REVIEW

File: api/src/controllers/listings.rs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        );
+    } else {
+        marketplace_account.unwrap()
+    };
+
+    // Send to market place
+    let marketplace_api = state.service_locator.create_marketplace_api()?;
+    let m_listing = marketplace_api.publish_listing(&listing, &marketplace_account)?;
+    listing.set_published(m_listing, conn)?;
+    unimplemented!()
```
Comment (dunnock) [inline api/src/controllers/listings.rs]: What about returning `Ok(HttpResponse::NotImplemented())` , with panic will be easy to DOS

File: sharetribe_flex/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+authors = ["Mike Berry <mikethetike@tari.com>"]
+edition = "2018"
+
+
+[dependencies]
+snafu = "0.6.2"
+chrono = {version="0.4.10", features=["serde"]}
+serde = "1.0.104"
+serde_json = "1.0.48"
+reqwest = "0.9"
```
Comment (dunnock) [inline sharetribe_flex/Cargo.toml]: What about using `"0.10"` and async?

**Comment by dunnock** [comment]
As discussed those might be upgraded after merging in separate PR

**Comment by dunnock** [comment]
@mikethetike this should fix integration tests: https://github.com/big-neon/bn-api/pull/1742

**Comment by dunnock** [comment]
One more fix for integrations https://github.com/big-neon/bn-api/pull/1744 .. it seems all issues down to the tests logic

**Comment by dunnock** [comment]
It seems POST `/users/me/marketplace_account` is still missing, do we need that integration test?

**Comment by mikethetike** [comment]
Will push something soon

**Pull request merged**